packages/fx-core/resource/package.nls.json

```diff
@@ -472,6 +472,8 @@
   "core.selectValidateMethodQuestion.validate.schemaOptionDescription": "Validate using manifest schema",
   "core.selectValidateMethodQuestion.validate.appPackageOption": "Validate app package using validation rules",
   "core.selectValidateMethodQuestion.validate.appPackageOptionDescription": "Validate app package using validation rules",
+  "core.selectValidateMethodQuestion.validate.testCasesOption": "Publish Readiness",
+  "core.selectValidateMethodQuestion.validate.testCasesOptionDescription": "Validate with test cases Microsoft uses when reviewing your app for publishing",
```

Comment:
Proposed: Check your app with the test cases Microsoft uses before they publish it.

Reply:
Thanks @supkasar , all resolved based on suggestions.